frontend/src/metabase/home/components/CustomHomePageModal/CustomHomePageModal.tsx

```diff
@@ -37,6 +40,28 @@ export const CustomHomePageModal = ({
         [CUSTOM_HOMEPAGE_REDIRECT_TOAST_KEY]: true,
       }),
     );
+
+    const id = Date.now();
+    await dispatch(
+      addUndo({
+        message: () => (
+          <Box ml="0.5rem" mr="2.5rem">
+            <Text
+              span
+              fw={700}
+            >{t`This dashboard has been set as your homepage.`}</Text>
+            <br />
+            {t`You can change this in Admin > Settings > General.`}
```

Comment:
Nit: is there a reason the second text was not put in a `<Text>` element as well?

```suggestion
            <Text 
              span
            >{t`You can change this in Admin > Settings > General.`}</Text>
```

Reply:
Not really. I only wrapped the other once because I needed to bold the text, but we could wrap this one as well.